### PR TITLE
Avoid Hazards

### PR DESCRIPTION
**Pull request by jessemarion**
https://docs.battlesnake.com/references/game-modes#royale

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -58,6 +58,7 @@ struct Board {
     height: isize,
     width: isize,
     food: Vec<Coord>,
+    hazards: Vec<Coord>,
```
Comment (mattcorbin) [inline src/main.rs:61]: does this get passed in in other modes? Might need to make this optional

File: src/moves.rs [hunk 2]
```diff
@@ -119,5 +119,28 @@ pub fn compute_move(game: &GameRequest) -> String {
             }
         }
     }
+
+    //  If the head of a Battlesnake is within the Hazard Sauce,
+    // its health points will more rapidly decline, loosing 15 health each turn.
```
Comment (mattcorbin) [inline src/moves.rs:124]: losing not loosing

**Review by mattcorbin — CHANGES_REQUESTED** [review]

File: src/moves.rs (hunk 2 above)
Comment (jessemarion) [reply src/moves.rs:124]: image

Comment (mattcorbin) [reply src/moves.rs:124]: just because they typed it wrong doesn't mean that you have an excuse to leave it wrong

File: src/main.rs (hunk 1 above)
Comment (jessemarion) [reply src/main.rs:61]: Not optional according to the api spec
https://docs.battlesnake.com/references/api#board

File: src/moves.rs (hunk 2 above)
Comment (jessemarion) [reply src/moves.rs:124]: It's unethical to misquote somebody and I have journalistic integrity

Comment (mattcorbin) [reply src/moves.rs:124]: I'm not going to approve until you fix the typo

Comment (jessemarion) [reply src/moves.rs:124]:
```suggestion
    // its health points will more rapidly decline, losing 15 health each turn.
```

Comment (mattcorbin) [reply src/moves.rs:124]: if i commit this can we even get this merged

**Review by mattcorbin — APPROVED** [review]

**Pull request merged**